entities/queries/ColumnQuery.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Entities\Queries;
+
+/**
+ * Query for column in a database.
+ */
+interface ColumnQuery
+{
+
+    /**
+     * Returns value of an entity column.
+     *
+     * @param string $column column name.
+     * @return string
+     */
+    public function result(string $column): string;
+}
```

Comment:
Interface for different queries. It is done so that entities do not depend on specific implementations.